src/components/builder.cpp

```diff
@@ -160,15 +161,15 @@ void builder::node(const label_t& label) {
     color_close();
   }
 
-  if (label->m_underline.has_color()) {
+  if (!label->m_underline.has_color() || (label->m_margin.right.value > 0 && m_tags[syntaxtag::u] > 0)) {
```

Comment:
```suggestion
  if (label->m_underline.has_color()) {
```

I don't really know what the second part is supposed to do.